package/nginx-http.conf

```diff
@@ -0,0 +1,11 @@
+server {
+    listen 80   default;
+    server_name rmt;
+    access_log  /var/log/nginx/rmt_http_access.log;
+    error_log   /var/log/nginx/rmt_http_error.log;
+    root        /usr/share/rmt/public;
+
```

Comment:
Why not white list the `tools/clientSetup4RMT.sh` only and redirect everything else to https?

Reply:
@felixsch What do you mean specifically? It doesn't do a redirect at the moment, but otherwise that's what it does.

@digitaltom That's up for discussion. SMT does allow access to RPMs over plain HTTP. If you want to learn how many users do that instead of using SUSECoonect -- we could disable access over HTTP and see how many users complain :grimacing:

Reply:
@digitaltom actually asked the question behind my statement. If no access to HTTP should be allowed, why not redirect to HTTPS everything except the `clientSetup4RMT.sh`.

Reply:
@kalabiyau Do you have any thoughts on this? Should the RPMs be accessible over plain HTTP?
SMT serves RPMs over HTTP, at the moment RMT does the same -- only the API is restricted to HTTPS.

Reply:
@ikapelyukhin as you described to me that atm we have:

registration would by default offer https:// urls
and plain access is allowed to the content via http://

I believe it is a sane setup.